comp/core/status/render_helpers.go

```diff
@@ -198,6 +199,13 @@ func parseUnixTime(value any) (time.Time, error) {
 		raw = v
 	case float64:
 		raw = int64(v)
+	// Case where the unix time is a time.Time and has been converted into a string date due to a JSON marshall
+	case string:
+		t, err := time.Parse(timeLayout, v)
```

Comment:
According to the documentation it uses format from RFC3339, so you could just use that ?https://pkg.go.dev/time#Time.MarshalJSON
```suggestion
		t, err := time.Parse(time.RFC3339, v)
```

Note that it's not quite the same format as you entered previously so maybe check that this works as expected first